system/ee/installer/views/success.php

```diff
@@ -1,4 +1,4 @@
-<div class="box <?=empty($update_notices) ? 'success' : 'warn'?>">
+<div class="box <?=empty($update_notices) || count($update_notices)==1 ? 'success' : 'warn'?>">
```

Comment:
@intoeetive should this be greater then or equal >= 1 ?

Reply:
It should not. We DO WANT to apply success style if there's just one notice, otherwise we apply 'warn' style

Reply:
What if there are 2 update notices?

Reply:
Previously, "please delete your install folder" was not part of `$update_notices` so the message was always shown as success. It is now part of `$update_notices`, so we want to make sure the screen is still green if there is just one message. If there's something more, we apply 'warn' style to indicate there are more thing to look into - just the way it was before

Reply:
Ah, ok cool, I think I follow now.